builtin/mainmenu/tab_credits.lua

```diff
@@ -63,27 +73,30 @@ local previous_core_developers = {
 }
 
 local previous_contributors = {
-	"Nils Dagsson Moskopp (erlehmann) <nils@dieweltistgarnichtso.net> [Minetest Logo]",
-	"Dániel Juhász (juhdanad) <juhdanad@gmail.com>",
-	"red-001 <red-001@outlook.ie>",
-	"numberZero [Audiovisuals: meshgen]",
+	-- TODO: add/keep areas of work here
+	"Nils Dagsson Moskopp",
+	"red-001",
 	"Giuseppe Bilotta",
-	"MirceaKitsune <mirceakitsune@gmail.com>",
-	"Constantin Wenger (SpeedProg)",
-	"Ciaran Gultnieks (CiaranG)",
-	"stujones11 [Android UX improvements]",
-	"Jeija <jeija@mesecons.net> [HTTP, particles]",
-	"Vincent Glize (Dumbeldor) [Cleanups, CSM APIs]",
-	"Ben Deutsch [Rendering, Fixes, SQLite auth]",
-	"TeTpaAka [Hand overriding, nametag colors]",
-	"Rui [Sound Pitch]",
-	"Duane Robertson <duane@duanerobertson.com> [MGValleys]",
-	"Raymoo [Tool Capabilities]",
-	"Rogier <rogier777@gmail.com> [Fixes]",
-	"Gregory Currie (gregorycu) [optimisation]",
-	"TriBlade9 <triblade9@mail.com> [Audiovisuals]",
-	"T4im [Profiler]",
-	"Jurgen Doser (doserj) <jurgen.doser@gmail.com>",
+	"numzero",
```

Comment:
I think "previous contributors" should exclude active ones to keep things shorter.

Reply:
Yes I merely forgot that.